docs/source/tutorial/installation-and-setup.rst

```diff
@@ -89,6 +89,8 @@ settings below, or use our :ref:`settings_example` for a complete
       'django.contrib.auth.backends.ModelBackend',
   ]
 
+  # Enable newest DGPF template styles
+  BASE_TEMPLATES = 'globus-portal-framework/v3/'
```

Comment:
I think it would be better to leave this out of tutorials and only as a reference config option, specified here:

https://django-globus-portal-framework.readthedocs.io/en/latest/reference/settings.html#templates